sdcm/nemesis.py

```diff
@@ -1866,6 +1867,34 @@ def modify_table_bloom_filter_fp_chance(self):  # pylint: disable=invalid-name
         """
         self._modify_table_property(name="bloom_filter_fp_chance", val=random.random() / 2)
 
+    def toggle_table_gc_mode(self):
+        """
+            Alters a non-system table tombstone_gc_mode option.
+            Choose the alternate option of 'repair' / 'timeout'
+             (*) The other available values of 'disabled' / 'immediate' are not tested by
```

Comment:
Please add a reference or describe what each GC mode means.

Reply:
[can be done in a different PR since already merged]

Reply:
please, add a PR with this information ASAP